packages/cms-cli/commands/customObject/schema/delete.js

```diff
@@ -12,12 +12,11 @@ const { setLogLevel, getPortalId } = require('../../../lib/commonOpts');
 const { logDebugInfo } = require('../../../lib/debugInfo');
 const { deleteSchema } = require('@hubspot/cms-lib/api/schema');
 
-exports.command = 'delete <schemaObjectType>';
-exports.describe =
-  'Delete a custom object schema given a schemaObjectType. Delete operation is asynchronous and may take time to complete even after the command succeeds';
```

Comment:
Removed the async part cuz in the success message I word it as "initiated deletion" which I hope is enough of a hint that deletion isn't instantaneous